kinto/plugins/openid/views.py

```diff
@@ -115,7 +115,7 @@ def get_login(request):
 
     # Redirect the client to the Identity Provider that will eventually redirect
     # to the OpenID token endpoint.
-    token_uri = request.route_url("openid_token", provider=provider) + "?"
+    token_uri = request.route_url("openid_token", provider=provider)
```

Comment:
Weird, I'm pretty it's here for a reason...  I'm afraid that if we remove it we break Auth0 or something...

Isn't there something you could configure in the Google settings?

Reply:
> Isn't there something you could configure in the Google settings?

Unfortunately nope.

Reply:
I can also add a settings, `add_trailing_question_mark` or detect that it is Google and remove it :confused:

Reply:
I am pretty sure Auth0 should add it automatically too.

Reply:
Could you try it on your end?

Reply:
> Could you try it on your end?

Ok I will on Kinto DEV

I know that I got this plugin working with Google ID :)

Reply:
> I know that I got this plugin working with Google ID :)

I do remember that, but it seems to me that they made some changes on their interface :confused: